t/property.t

```diff
@@ -0,0 +1,35 @@
+use t::WASM 'no_plan';
+
+run_tests();
+
+__DATA__
+
+=== TEST 1: load
+--- config
+location /t {
+    content_by_lua_block {
+        local wasm = require("resty.proxy-wasm")
+        assert(wasm.load("plugin", "t/testdata/property/main.go.wasm"))
```

Comment:
We don't need to test loading in the property test file.

Reply:
removed